# HG changeset patch
# User Rik <rik@octave.org>
# Date 1549644456 28800
#      Fri Feb 08 08:47:36 2019 -0800
# Node ID 8e97a2e5ab665e3c1c8b01ec2c4bcd16d77d9a29
# Parent  fa56924691a9cb0f6ae2c7526ac0651f10ff6912
Read sitewide init-file when running 'make check'

* test/module.mk: Change invocation of octave to use '--no-init-file' so that
only the user's personal octaverc files are skipped.

diff --git a/test/module.mk b/test/module.mk
--- a/test/module.mk
+++ b/test/module.mk
@@ -83,17 +83,17 @@ include %reldir%/ctor-vs-method/module.m
 include %reldir%/fcn-handle-derived-resolution/module.mk
 include %reldir%/local-functions/module.mk
 include %reldir%/mex/module.mk
 include %reldir%/nest/module.mk
 include %reldir%/publish/module.mk
 include %reldir%/pkg/module.mk
 
 define run-octave-tests
-  ( cd %reldir% && $(SHELL) ../run-octave $(RUN_OCTAVE_OPTIONS) $(1) --norc --silent --no-history -p $(abs_top_builddir)/%reldir%/mex $(abs_top_srcdir)/%reldir%/fntests.m $(abs_top_srcdir)/%reldir% ) && \
+  ( cd %reldir% && $(SHELL) ../run-octave $(RUN_OCTAVE_OPTIONS) $(1) --no-init-file --silent --no-history -p $(abs_top_builddir)/%reldir%/mex $(abs_top_srcdir)/%reldir%/fntests.m $(abs_top_srcdir)/%reldir% ) && \
   if $(AM_V_P); then \
     echo ""; \
     if [ -f %reldir%/fntests.log ]; then \
       echo "Contents of %reldir%/fntests.log:"; \
       echo ""; \
       $(AWK) -f $(srcdir)/%reldir%/show-failures.awk %reldir%/fntests.log; \
     else \
       echo "%reldir%/fntests.log is missing!"; \
